packages/apps-config/src/endpoints/testingRelayRococo.ts

```diff
@@ -242,6 +242,14 @@ export function createRococo (t: TFunction): EndpointOption {
           'Konomi Network': 'wss://rococo.konomi.tech'
         }
       },
+      {
+        info: 'rococoKylin',
+        paraId: 2000,
```

Comment:
One last question - is this indeed yours? ;)

Reply:
Def, I own this account and I just registered the parathread to this account
`5Gn1igfpf4hP7iG1Gsm1AbwPBCpR8BmHK4b6i2VrGHQS1kAJ`


![image](https://user-images.githubusercontent.com/4311963/119187853-c8ed5580-ba71-11eb-9e35-7e08d3c71514.png)

Reply:
No issues, always like "2000" - will always be first :)